cms/src/collections/collections.ts

```diff
@@ -30,8 +30,8 @@ export const contributors: CmsCollection = {
   files: [
     {
       label: 'All Contributors',
-      name: '.all-contributorsrc',
-      file: '/.all-contributorsrc',
+      name: 'all-contributorsrc',
+      file: '/all-contributorsrc',
```

Comment:
The file name must be `.all-contributorssrc`. But the `name` field in cms config must be without dot i.e. `all-contributorsrc`. So don't rename the file, only remove the dot from the name. Like so... :)
https://github.com/29ki/29k/blob/4b9462b8ff958179cd380586af39daed07ba7f8e/cms/src/index.ts#L78-L81

Reply:
Ok, the name was without . in my branch, some weird state here? Anyways, reverted the filename now